app-api/src/main/java/org/openmrs/eip/app/management/entity/SyncResponse.java

```diff
@@ -0,0 +1,80 @@
+package org.openmrs.eip.app.management.entity;
+
+import java.util.Date;
+
+import javax.persistence.Access;
+import javax.persistence.AccessType;
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.EnumType;
+import javax.persistence.Enumerated;
+import javax.persistence.Table;
+import javax.validation.constraints.NotNull;
+
+@Entity
+@Table(name = "sync_response")
+public class SyncResponse extends AbstractEntity {
+	
+	public static final long serialVersionUID = 1;
+	
+	public enum SyncResponseStatus {
+		NEW,
+		PROCESSED
+	}
+	
+	@NotNull
+	@Column(name = "message_uuid", length = 38, nullable = false, unique = true, updatable = false)
+	@Access(AccessType.FIELD)
+	private String messageUuid;
+	
+	@NotNull
+	@Column(name = "date_sent", precision = 3, nullable = false, updatable = false)
+	@Access(AccessType.FIELD)
+	private Date dateSent;
+	
+	@Column(name = "date_processed", precision = 3)
+	@Access(AccessType.FIELD)
+	private Date dateProcessed;
+	
+	@NotNull
+	@Enumerated(EnumType.STRING)
+	@Column(nullable = false, length = 10)
+	@Access(AccessType.FIELD)
+	private SyncResponseStatus status;
+	
+	public SyncResponse(@NotNull String messageUuid, @NotNull Date dateSent) {
```

Comment:
This being a persistent class, you need to arg constructor

Reply:
Do you use this other constructor anywhere? If not, you might as well get rid of it

Reply:
@lccpinto did you see this comment?

Reply:
Sorry @wluyima, I didn't see because it was marked as "resolved" and I was hiding the resolved chats.
I was using this constructor, but I did a refactor to not use a constructor at all, to be aligned with the rest of the code.